src/transform/ElementUtilities.js

```diff
@@ -44,6 +46,7 @@ const isNestedInTable = (el) => {
 }
 
 export default {
+  matchesSelectorCompat,
```

Comment:
I can ElementUtilities is already under test though. This would end up looking like `pagelib.test.ElementUtilities.test.matchesSelectorCompat`.